subworkflows/ensembl/prepare_genome_metadata/meta.yml

```
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "prepare_genome_metadata"
+description: A subworkflow to query specific genome metadata from a full genome summary report JSON \
+  obtained via NCBI datasets-cli.
+keywords:
+  - ensembl
+  - genome metadata
+  - genomio
+  - JSON
+  - metadata
+  - utility
+  - validation
+components:
+  - Ensembl/genomio:genome_metadata_prepare
+  - Ensembl/genomio:schemas_json_validate
```

Not sure what you are referring to exactly here, if Python, then `ensembl-genomio` is the right approach, if GitHub, maybe `Ensembl/ensembl-genomio`:
```suggestion
  - ensembl-genomio:genome_metadata_prepare
  - ensembl-genomio:schemas_json_validate
```